src/mapped-types.ts

```diff
@@ -6,24 +6,40 @@
 /**
  * SetIntersection (same as Extract)
  * @desc Set intersection of given union types `A` and `B`
+ * @example
+ *   // Expect: "2" | "3"
+ *   type ResultSet = SetIntersection<'1' | '2' | '3', '2' | '3' | '4'>;
+ *   // Expect: () => void
```

Comment:
Please add 1 empty line between examples for better readability. Please follow up this convention in all the examples below

Reply:
done,